### ISSUE TITLE
RuntimeError when run test_tracktor.py

### ISSUE THREAD

**Issue by sfwang20** [phil-bergmann/tracking_wo_bnw#143]
Hi Philipp, 
Thanks for your excellent work!
I followed README.md to build the environment and when I run test_tracktor.py, it shows the RuntimeError as following:
> INFO - main - Initializing object detector(s).
> INFO - main - Initializing reID network(s).
> INFO - main - Tracking: MOT17-02-FRCNN
>   0%|                                                                                                                              | 0/600 [00:02<?, ?it/s]
> ERROR - test_tracktor - Failed after 0:00:27!
> Traceback (most recent calls WITHOUT Sacred internals):
>   File "experiments/scripts/test_tracktor.py", line 143, in main
>     tracker.step(frame_data)
>   File "d:\research\projects\tracking_wo_bnw\src\tracktor\tracker.py", line 261, in step
>     self.obj_detect.load_image(blob['img'])
>   File "d:\research\projects\tracking_wo_bnw\src\tracktor\frcnn_fpn.py", line 75, in load_image
>     self.features = self.backbone(preprocessed_images.tensors)
>   File "C:\Users\shinfu\anaconda3\envs\tracking_wo_bnw\lib\site-packages\torch\nn\modules\module.py", line 722, in _call_impl
>     result = self.forward(*input, **kwargs)
>   File "C:\Users\shinfu\anaconda3\envs\tracking_wo_bnw\lib\site-packages\torchvision\models\detection\backbone_utils.py", line 39, in forward
>     x = self.body(x)
>   File "C:\Users\shinfu\anaconda3\envs\tracking_wo_bnw\lib\site-packages\torch\nn\modules\module.py", line 722, in _call_impl
>     result = self.forward(*input, **kwargs)
>   File "C:\Users\shinfu\anaconda3\envs\tracking_wo_bnw\lib\site-packages\torchvision\models\_utils.py", line 63, in forward
>     x = module(x)
>   File "C:\Users\shinfu\anaconda3\envs\tracking_wo_bnw\lib\site-packages\torch\nn\modules\module.py", line 722, in _call_impl
>     result = self.forward(*input, **kwargs)
>   File "C:\Users\shinfu\anaconda3\envs\tracking_wo_bnw\lib\site-packages\torch\nn\modules\conv.py", line 419, in forward
>     return self._conv_forward(input, self.weight)
>   File "C:\Users\shinfu\anaconda3\envs\tracking_wo_bnw\lib\site-packages\torch\nn\modules\conv.py", line 416, in _conv_forward
>     self.padding, self.dilation, self.groups)
> RuntimeError: cuDNN error: CUDNN_STATUS_ALLOC_FAILED

I search by google if there are solutions about the error and it seems like the memory of my GPU is not enough.
My GPU is Nvidia GTX 1660Ti with 6 GB memory.
Besides, I also tried change the batch size (to 32) in res50.yml but it didn't work (it seems like testing won't use this file, I'm not sure about that)
Is there any other way for me to get smaller GPU memory usage?
Thanks for your help!
Best regards.

**Comment by timmeinhardt** [comment]
Yes, the file you changed is just for the training of the reid model. You could run the tracking evaluation on the CPU or reduce the resolution of the images. The latter might hurt the performance though.
